PC-VRPTW/RC1_10_8.vrp

```diff
@@ -0,0 +1,4019 @@
+NAME : RC1_10_8
+TYPE : CVRPTW
+DIMENSION : 1001
+VEHICLES : 90
+CAPACITY : 200
+SERVICE_TIME : 10
+EDGE_WEIGHT_TYPE : EUC_2D
```

Comment:
Question: What is going to be the rounding convention for these instances? Do we assume integer rounding or DIMACS rounding?

Reply:
The BKS results were obtained using the DIMACS conventions. It's probably good to add that to the README, one sec.